source/Octopus.Client/LocationChecker.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Win32;
+using Octopus.Client.Logging;
+
+namespace Octopus.Client
+{
+#if FULL_FRAMEWORK
+    class LocationChecker
+    {
+        private static readonly ILog Logger = LogProvider.For<LocationChecker>();
+
+        public static void CheckAssemblyLocation()
+        {
+            var octopusRegNode = Registry.LocalMachine.OpenSubKey(@"Software\Octopus");
```

Comment:
I think `octopusRegNode` instance is `IDisposable`?

Reply:
Indeed it is, good 👀 ! Fixed, and the other one below too